### PR TITLE
Add sdk configuration hooks for development purposes

### LINKED ISSUE #320: Expose a way to configure sdk during local development
When we are running local version of opint alongside local version of gearbox (via `yarn link`), gearbox uses a version of the sdk that is not configured, so all api requests made from its components fail, which greatly limits local development of opint using gearbox 🙁.

It doesn't look like opint currently uses any gearbox components in such a way that the gearbox component makes api requests directly, but we'll need to do _some_ thing to solve this when we do...

When we ran into this issue with locally developing griff-loader, we solved it by exposing a method to configure griff-loader's sdk through opint. [Code here](https://github.com/cognitedata/griff-loader/pull/2/files#diff-1fdf421c05c1140f6d71444ea2b27638R12), [usage here](https://github.com/cognitedata/operational-intelligence/blob/staging/src/utils/Environment.js#L113).

Maybe there is a better way to solve this now though. Feel free to suggest.

### PR DESCRIPTION
**Pull request by mrtraser**
Fixes #320

### CODE REVIEW

**Comment by mrtraser** [comment]
@stonecrafter , should I provide additional info on how to use it in the development section of README.md?

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-349.gearboxjs.preview.cogniteapp.com

**Comment by stonecrafter** [comment]
Yes, please do add some documentation about usage! Thanks! 😄

File: README.md [hunk 1]
```diff
@@ -38,8 +38,35 @@ See the up-to-date storybook [here](https://cognitedata.github.io/gearbox.js).
...
+  project: 'project_name',
+  ...
+})
+```
+
+or you can run through the login process passing tenant name like below:
+```typescript
+import { authSdk } from '@cognite/gearbox'
+
+async function loginIn(tenant: string) {
```
Comment (stonecrafter) [inline README.md]:
```suggestion
async function login(tenant: string) {
```

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-349.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-349.gearboxjs.preview.cogniteapp.com

**Pull request merged**